src/overlays/actors/ovl_En_Weiyer/z_en_weiyer.c

```diff
@@ -52,59 +63,590 @@ static ColliderCylinderInit D_80B33A90 = {
+
+    if (Math_ScaledStepToS(&this->actor.shape.rot.y, this->unk_196, 182)) {
+        if (this->unk_194 != 0) {
+            this->unk_194--;
+        }
+
+        if (this->unk_194 == 0) {
+            this->unk_196 =
+                Rand_S16Offset(0x2000, 0x2000) * ((Rand_ZeroOne() < 0.5f) ? -1 : 1) + this->actor.shape.rot.y;
+            this->unk_194 = 30;
+
+            if (Rand_ZeroOne() < 0.3333f) {
+                Audio_PlayActorSound2(&this->actor, NA_SE_EN_EIER_CRY);
+            }
+        }
+    }
+
+    if (this->actor.home.pos.y < this->actor.world.pos.y) {
+        if (this->actor.home.pos.y < this->actor.floorHeight) {
+            func_80B32434(this);
+        } else {
+            this->actor.world.pos.y = this->actor.home.pos.y;
+            this->unk_280 =
+                Rand_ZeroOne() * ((this->actor.home.pos.y - this->actor.floorHeight) / 2.0f) + this->actor.floorHeight;
+        }
+    } else {
+        Player* player = PLAYER;
+
+        if (this->actor.bgCheckFlags & 1) {
+            this->unk_280 =
+                this->actor.home.pos.y - Rand_ZeroOne() * ((this->actor.home.pos.y - this->actor.floorHeight) / 2.0f);
+        } else if (sp34 && (Rand_ZeroOne() < 0.1f)) {
+            this->unk_280 =
+                Rand_ZeroOne() * (this->actor.home.pos.y - this->actor.floorHeight) + this->actor.floorHeight;
+        }
+
+        if ((this->actor.xzDistToPlayer < 400.0f) && (fabsf(this->actor.yDistToPlayer) < 250.0f) &&
+            (player->actor.world.pos.y < (this->actor.home.pos.y + 20.0f))) {
+            func_80B32508(this);
+        }
+    }
+}
+
+void func_80B32C2C(EnWeiyer* this, GlobalContext* globalCtx) {
+    SkelAnime_Update(&this->skelAnime);
+
+    if (this->unk_194 == 0) {
+        if (Math_ScaledStepToS(&this->actor.shape.rot.x, -0x4000, 0x800)) {
+            this->actor.shape.rot.z = 0;
+            this->actor.shape.rot.y += 0x8000;
+            this->unk_194 = 1;
+        } else {
+            this->actor.shape.rot.z = this->actor.shape.rot.x * 2;
+        }
+    } else {
+        Math_ScaledStepToS(&this->actor.shape.rot.x, 0x1800, 0x800);
+
+        if (this->actor.world.pos.y < this->actor.home.pos.y) {
+            if (this->actor.shape.rot.x > 0) {
+                EffectSsGSplash_Spawn(globalCtx, &this->actor.world.pos, NULL, NULL, 1, 0x190);
+                Audio_PlayActorSound2(&this->actor, NA_SE_EN_OCTAROCK_SINK);
+            }
+
+            func_80B32538(this);
+        } else if (this->actor.bgCheckFlags & 1) {
+            func_80B32494(this);
+        }
+    }
+}
+
+void func_80B32D30(EnWeiyer* this, GlobalContext* globalCtx) {
+    SkelAnime_Update(&this->skelAnime);
+
+    if (Animation_OnFrame(&this->skelAnime, 0.0f)) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_EIER_FLUTTER);
+    }
+
+    Math_ScaledStepToS(&this->actor.shape.rot.x, 0, 0x800);
+    Math_StepToF(&this->actor.speedXZ, 0.0f, 1.0f);
+
+    if (this->unk_194 != 0) {
+        this->unk_194--;
+    }
```

Comment:
```suggestion
    DECR(this->unk_194);
```

Reply:
there are a few more instances of these with `unk_194`, but I don't feel like marking all of them.